client/src/components/DropdownMenu/Dropdown.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './DropdownStyles.css';
+
+const Dropdown = ({ people }) => {
+  return (
+    <div class="dropdown">
+      <div href="#" class="dropbtn">
+        Contact
+      </div>
+      <div class="dropdown-content">
+        {people.map((person) => {
+          return (
+            <li>
```

Comment:
need to supply the key attribute because currently this causes an error in the console.
`key={index}` should be fine here